samples/cellular/lwm2m_client/sample_description.rst

```diff
@@ -397,9 +407,140 @@ CONFIG_SENSOR_MODULE_LIGHT - Configuration for light reading
 CONFIG_SENSOR_MODULE_COLOR - Configuration for color
    This configuration option enables the reading of color values.
 
+.. _lwm2m_configuration_files:
+
+Configuration files
```

Comment:
According to the[ sample template](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html#configuration-files), Configuration files should come after Additional configuration.

Reply:
We was gone trough with @SeppoTakalo and it make sense in that order.

Reply:
it is softer for reader until jump to long kconfig list